pkg/networkmanager/firewall.jsx

```diff
@@ -836,7 +836,7 @@ class ActivateZoneModal extends React.Component {
                                 { customZones.map(z =>
                                     <Radio key={z} id={z} name="zone" value={z}
                                            isChecked={this.state.zone == z}
-                                           onChange={(value, e) => this.onChange("zone", e.target.value)}
+                                           onChange={e => this.onChange("zone", e.target.value)}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230502-115030-f91148b7-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)